src/components/displaySubjects/ChapterImage.tsx

```diff
@@ -0,0 +1,49 @@
+import { FC, useState } from "react";
+import { Chapter } from "../../common/courseConstants";
+import "./SelectChapter.css";
+import Course from "../../models/course";
+import Lesson from "../../models/lesson";
+import { url } from "inspector";
+import CachedImage from "../common/CachedImage";
+
+const ChapterImage: FC<{
+  course: Course;
+  chapter: Chapter;
+}> = ({ course, chapter }) => {
+  const [count, setCount] = useState(1);
+
+  return (
+    <div className="chapter-icon">
```

Comment:
Create 1 custom img component for LessonImage and ChapterImage with this functionality and use it in LessonImage and ChapterImage